distributed/tests/test_client.py

```diff
@@ -6600,6 +6600,35 @@ def log_scheduler(dask_scheduler):
     assert events[1][1] == ("alice", "bob")
 
 
+@gen_cluster(client=True, nthreads=[])
```

Comment:
It looks like a bunch of `+` at the beginnings of the lines got in here when applying the suggestion which is causing CI to fail.

Reply:
It was an indentation issue, I've fixed it